api/entry.ml

```diff
@@ -20,48 +20,3 @@ type entry =
   | DTree of loc * mident option * ident
   | Name  of loc * mident
   | Require of loc * mident
-
-let pp_entry fmt e =
```

Comment:
Why are you suppressing `Entry.pp_entry` in this PR. Does it has anything to do with Public/Private Symbols?

Reply:
No, indeed. However, I noticed that this function was redundant with the one in `Pp`. This also makes the diff *negative* and I suspect this will please @Gaspi .